packages/babel-parser/test/fixtures/esprima/invalid-syntax/migrated_0222/output.json

```diff
@@ -13,8 +13,7 @@
     }
   },
   "errors": [
-    "SyntaxError: Octal literal in strict mode (1:36)",
-    "SyntaxError: Octal literal in strict mode (1:57)"
```

Comment:
Need to figure out why this second error was removed, as there are two octal literals in this test.

Reply:
<details>
<summary> :warning: Spoiler :warning: </summary>

This is probably because before this PR, we first checked if there was a `use strict` directive _and then_ tokenized the invalid literals.
Now, we first tokenize the invalid literals, then set `this.state.strict` to `true`, and in `parseBlockOrModuleBlockBody` only throw for the first invalid one. It should be fixed there.

</details>

Reply:
This has now been fixed.